components/organisms/PokemonCard/PokemonCard.tsx

```diff
@@ -3,14 +3,11 @@ import React from "react";
 import styles from "./PokemonCard.module.css";
 import { CardLink } from "../../molecules";
 import { POKEMON } from "../../../config";
+import { Pokemon } from "@generated";
```

Comment:
Hey look at that! Auto generated `Pokemon` type, which is **exactly** how it's defined on the GraphQL API. No overhead for us to maintain consistency with the API.